### PR TITLE
Validation Result Common passed (EXPOSUREAPP-8031)

### PR DESCRIPTION
**Pull request by BMItr**
This PR addresses ExposureApp-8031 (valid result screen).

Description coming...

how to test:
- Scan a valid TestCert
- Click "Gültigkeit prüfen" inside the cert view
- check the result (failed, valid, etc.)
Note: as it is development state, you can set  _get() = State.PASSED_ (line 24) inside the _DCCValidation.kt_ file to force this screen.

Screenshots:

vaild2
(scrolled a little bit down)
valid1
dark1

- [x] Missing Darkmode elements

### CODE REVIEW

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

**Review by mtwalli — COMMENTED** [review]
Lgtm 
Text size for the following text should be bigger I think `14 sp`
Screenshot 2021-07-09 at 16 03 30

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/validationresult/passed/DccValidationPassedFragment.kt [hunk 1]
```diff
@@ -1,5 +1,73 @@
...
+                    addItemDecoration(
+                        RecylerViewPaddingDecorator(
+                            topPadding = padding,
+                            leftPadding = padding,
+                            rightPadding = padding
+                        )
+                    )
+                }
+
+                toolbar.setOnClickListener { viewModel.onCloseClicked() }
```
[the remark this answers is not in the document]
Comment (BMItr) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/validationresult/passed/DccValidationPassedFragment.kt]: thx for the hint.
adjusted

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Review by mtwalli — APPROVED** [review]

**Pull request merged**